e2e/portalicious/tests/RegistrationPage/ViewRegistrationSummary.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { expect, test } from '@playwright/test';
+
+import { SeedScript } from '@121-service/src/scripts/enum/seed-script.enum';
+import {
+  getRegistrationIdByReferenceId,
+  seedPaidRegistrations,
+} from '@121-service/test/helpers/registration.helper';
+import {
+  getAccessToken,
+  resetDB,
+} from '@121-service/test/helpers/utility.helper';
+import {
+  programIdPV,
+  registrationPvScoped,
+} from '@121-service/test/registrations/pagination/pagination-data';
+
+import LoginPage from '@121-e2e/portalicious/pages/LoginPage';
+import RegistrationActivityLogPage from '@121-e2e/portalicious/pages/RegistrationActivityLogPage';
+
+const projectId = 2;
+let registrationId: number;
+
+test.beforeAll(async ({}) => {
+  await resetDB(SeedScript.nlrcMultiple);
+
+  const accessToken = await getAccessToken();
+  await seedPaidRegistrations([registrationPvScoped], projectId);
+  registrationId = await getRegistrationIdByReferenceId({
+    programId: programIdPV,
+    referenceId: registrationPvScoped.referenceId,
+    accessToken,
+  });
+});
+
+test('User should see a summary of a registration', async ({ page }) => {
```

Comment:
If these would be api tests I would have seperated each test step to a seperate 'test' and would have done the login part in the before all step. However that is not really straight forward here so decide to have one test with multiple steps to save a bit of processing time